src/app/components/Promo/image.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import ImagePlaceholder from '@bbc/psammead-image-placeholder';
+import { string } from 'prop-types';
+import { GEL_SPACING } from '@bbc/gel-foundations/spacings';
+
+const Img = styled.img`
+  width: 100%;
+  object-fit: cover;
+`;
+
+const Wrapper = styled.div`
+  margin-bottom: ${GEL_SPACING};
+`;
+
+// TODO: media indicators, srcsets and webp
+const Image = props => {
+  return (
+    <Wrapper>
+      <ImagePlaceholder ratio={56.25}>
```

Comment:
Where does this ratio number come from?

Reply:
This is 9/16, which we then multiply by 100 because this image placeholder wants it as a percentage.

We use this landscape 9:16 ratio for a lot of our images - it seems to be getting used in the topic design too - https://app.zeplin.io/project/5d4d77b1fcbfc79bf47e0294/screen/62027be867b5460425f8f0b5